_posts/2016-01-20-release-add-ons-1.8.md

```diff
@@ -0,0 +1,129 @@
+---
+title:  "Release: Add-Ons 1.8"
+---
+
+We are proud to announce the availability of Add-On Self Administration 1.8 and Add-On Administration 1.8.
+This release mostly brings compatibility fixes to make the add-ons work with all versions of OSIAM >= 2.2 again.
+Support for OSIAM 3.0 has been added too.
+Also a major bug has been fixed in the Add-On Administration.
```

Comment:
That's just a matter of taste but I don't like starting sentences with "Also" I would say: A major bug in Add-On Administration was fixed as well.